docs/core/fees.md

```diff
@@ -211,28 +377,105 @@ You could then use this data to estimate an appropriate prioritization fee for
 your transaction to both (a) better ensure it gets processed by the cluster and
 (b) minimize the fees paid.
 
-## Fee Collection
+## Rent
+
+The fee deposited into every [Solana Account](/docs/core/accounts.md) to keep
+its associated data available on-chain is called "_rent_". This fee is withheld
+in the normal lamport balance on every account and reclaimable when the account
+is closed.
+
+> Rent is different from [transactions fees](#transaction-fees). Rent is "paid"
+> (withheld in an Account) to keep data stored on the Solana blockchain and can
+> be reclaimed. Whereas transaction fees are paid to process
+> [instructions](/docs/core/transactions.md#instructions) on the network.
+
+All accounts are required to maintain a high enough lamport balance (relative to
+its allocated space) to become [rent exempt](#rent-exempt) and remain on the
```

Comment:
As much as I would loooooove to remove it and stop calling it rent, I specifically added info about it since all of the RPC methods and therefore code snippets use the term "rent exempt". I think if we removed it from the docs, it would add confusion and friction for devs since they WILL run into the terms as soon as they go to write code

Reply:
Alright we keep for now.